<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook ms9">2.036<span class="authors"></span></div>
</div></header>


<footer class="navbar navbar-default navbar-fixed-bottom mcs-psy-nav" >
  <div class="container-fluid text-right h4">
    <em>ψᵟ</em>
  </div>
</footer>




## Given 

As shown in the figure, an oven wall consists of a 0.635 cm thick layer of steel ($\LatoMath{κ_s}$ = 15.1 W/m·K) and a layer of brick ($\LatoMath{κ_b}$ = 0.72 W/m·K). At steady state, a temperature decrease of 0.7 °C occurs over the steel layer. The inner temperature of the steel layer is 300 °C. <br><br><br>


![Fig. P2.36](../figures/ms9,2-036⋮w0164.png)


## Find

If the temperature of the outer surface of the brick must not exceed 40 °C,

1. Determine the thickness of brick, in cm, that ensures this limit is met.  
1. What is the rate of conduction, in kW per m² of wall surface area?


### Get Organized

<div class="mcs-bs-callout mcs-bs-callout-info">

<h4> Before you dive in </h4>
  
Here is a quick Heat Transfer primer:  
[Just enough heat transfer to make you dangerous](enough-heat-transfer-to-make-you-dangerous.jl.ipynb)

</div>

When resolving the details of any heat transfer scenario, always look for these 3 elements:

1. a temperature difference;
1. some factor (often the product of multiple terms) detailing the heat transfer mechanism; and
1. of course, the heat flow itself.

<b class="mcs-primary">The key to this problem is recognizing that all of the heat flowing through the steel also flows through the bricks. It's the **only** path the heat has to escape the oven. </b>

So express your base thermodynamic relationship.  

\begin{gather*} 
\dot Q_\textit{steel} = \dot Q_\textit{brick} \\{}
\end{gather*}

Add details germane to our heat transfer mode, conduction.

\begin{gather*} {}\\
\dot Q_\textit{steel} = \left. kA\frac{\Delta T}{\Delta L} \right|_\textit{steel} = \left. kA\frac{\Delta T}{\Delta L} \right|_\textit{brick} = \dot Q_\textit{brick} \\{}
\end{gather*}

We're asked to find heat flux ($\dot Q/A$), and the area is the same on both sides of the equation, so we can safely divide it out.

\begin{gather*} {}\\
q''_\textit{steel} 
= \left. k\frac{\Delta T}{\Delta L} \right|_\textit{steel} 
= \left. k\frac{\Delta T}{\Delta L} \right|_\textit{brick} 
= q''_\textit{brick} \\{}
\end{gather*}

Switch from thermodynamics to math and solve for the requested information.

\begin{align*} \\{}
q'' &=  \pu{15.1 W/m·K}\frac{\pu{0.7 K^{*}}}{\pu{0.635cm}} \\[9pt]
    &= \pu{0.72 W/m·K} \frac{\pu{(300 - 0.7)K^{*} - 40 K^{*}} }{\Delta L_\textit{brick}} 
\end{align*}

\begin{multline*}
\shoveright \LatoText{*Remember, Δ°C = ΔK}
\end{multline*}

## Attack

Put Julia to work. 

In [1]:
using Unitful

ks  = 15.1u"W/(m*K)"
ΔTs = 0.7u"K"
ΔLs = 0.635u"cm"

kb  = 0.72u"W/(m*K)"
ΔTb = (300 - 0.7 - 40)u"K"

qʺ  = ks * ΔTs / ΔLs |> u"kW/m^2"; @show qʺ
ΔLb = kb * ΔTb / qʺ  |> u"cm";     @show ΔLb;

qʺ = 1.6645669291338583 kW m^-2
ΔLb = 11.215890255439925 cm


<div class="alert alert-success" role="alert" style="border: 1px solid #c3e6cb;">
<H2> Found</h2>
  
1. Minimum brick thickness $\LatoMath{= 11.22\, cm}$
  
1. Heat flux $~\LatoMath{= 1.665\, kW/m^2}$

</div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>